doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -332,11 +332,12 @@ Bluetooth samples
 
   * Fixed an interoperability issue with iOS devices by setting the report IDs of HID input and output reports to zero.
 
-* :ref:`peripheral_fast_pair` sample:
+* :ref:`fast_pair_input_device` sample:
```

Comment:
you may need to align the name for the past release notes (release notes 2.0.0, 2.1.0, 2.2.0, 2.4.0)

Reply:
Ok, I was wondering if I should do it or not.

Reply:
I am not sure if we shouldn't use the label value here (that is "Bluetooth: Fast Pair"). @divipillai ?

Reply:
We provide the sample name in old release notes. I have added it as a suggestion for clarity.